webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/xmlparsers/TechniqueParser.scala

```diff
@@ -85,7 +85,7 @@ class TechniqueParser(
             policyTypes        = ((xml \ TECHNIQUE_POLICY_TYPES)) match {
                                    case n if (n.isEmpty) => PolicyTypes.compat(isSystem)
                                    case n                =>
-                                     n.text.split(",").toList match {
+                                     n.flatMap(_.text.split(",").toList).toList match {
```

Comment:
Was accepting several  policytypes entries but mixed them, now we support both having multiple entries and entries with commas